test/dotnet-migrate.Tests/GivenThatIWantToMigrateTestApps.cs

```diff
@@ -121,6 +121,7 @@ public void It_migrates_an_app_with_scripts_and_the_scripts_run()
         private MigratedBuildComparisonData GetDotnetNewComparisonData(string projectDirectory, string dotnetNewType)
         {
             DotnetNew(projectDirectory, dotnetNewType);
+            File.Copy("NuGet.tempaspnetpatch.config", Path.Combine(projectDirectory, "NuGet.Config"));
```

Comment:
Actually, this needs to happen between the new and the restore. So, this is the best place to do it. Unless we have a reason to move it out. I would leave it here.